scripts/CI_scripts/check_multitest.sh

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env bash
+
+set -e
```

Comment:
We need to get notification in case of any error in this script. Thus better to remove `set -e` and use for loop over ("astroport-core", "astroport-governance", "astroport-bootstrapping") where you need to check return code of each check_repository and send whole std/stderr via slack.

Reply:
Sure, but on the last meet call,  we talk about and decided to leave it in the Astroport-core repository.

Reply:
Without set -e, it's possible that, for e. g. when pulling fails, we don't get any errors due to passed tests of out-of-date code. Using set -e, calling this script can send slack message on command errors too